guides/micronaut-rest-assured/micronaut-rest-assured.adoc

```diff
@@ -0,0 +1,40 @@
+common:header.adoc[]
+
+== REST-assured
+
+In this guide, you test a Micronaut application using https://rest-assured.io[REST-assured]
```

Comment:
"you test" -> "you will test"

Reply:
resolved via https://github.com/micronaut-projects/micronaut-guides/pull/900/commits/07303717ab9b664add0b9e8526f562dd6123103e